### ISSUE TITLE
如何点击buttonplus不展开菜单？

### ISSUE THREAD

**Issue by sendtion** [550609334/FloatingActionButtonPlus#6]
我只想要一个按钮，点击跳转到另一个Activity，滑动隐藏和显示。
现在，我点击按钮后，虽然没有菜单项，但是还是展开了，请问怎么解决？
我看源码里面open和close没有开放。

**Comment by liuzipengWally** [comment]
是的 我并没有将主FAB的点击事件放出接口，因为无论是 跟随滑动隐藏显示  还是 单击事件  仅仅一个普通的fab就可以做到了啊。

**Comment by sendtion** [comment]
@550609334 我已经用Android自带的实现了。我只是想，有的界面需要一个按钮就可以，有的界面需要弹出菜单，如果都用你的控件效果会比较统一，还是建议放开这个接口。
